### DISCUSSION TITLE
Job Status Stuck - Cannot Clear

### DISCUSSION THREAD

**Discussion by MikNolhan** [immich-app/immich#13364] [Q&A]
### The bug

Suddenly immich just stopped working. No changes at all, nothing, but it just processing new thumbnails stating it could not connect to postgres. Then It worked fine, and now the status does not update thumbnail counts, and logs do not generate even when set to verbose. They show the status changes, but nothing else. It seems it might be limited to this user account though.

![Screenshot 2024-10-10 at 17 20 26](https://github.com/user-attachments/assets/41e4d008-7913-47cd-b68f-f4b96fd6d0c1)

### The OS that Immich Server is running on

Debian

### Version of Immich Server

v1.117.0

### Version of Immich Mobile App

v1.117.0

### Platform with the issue

- [X] Server
- [X] Web
- [ ] Mobile

### Your docker-compose.yml content

```YAML
NA Runing Naively
```

### Your .env content

```Shell
N/A Running Natively
```

### Reproduction steps

1. Issue started on its own. Changing job concurrency, restarting services, PSQL, or computer has no impact

### Relevant log output

```shell
Logs are not working, they do not generate suddently.
```

**Comment by mmomjian** [comment]
You will need to post logs or some details about what is going on so someone can help. Might be better asked on the Discord as well for debugging

**Reply by MikNolhan** [reply]
I have no logs generating to post.

**Reply by MikNolhan** [reply]
What is weird is that it seems 25 is the new 0. I guess the question is how do I reset that count? Or what PSQL command can I run to resolve that.

**Reply by mmomjian** [reply]
That doesn't make sense, how do you have no logs? what shows up when you view the logs?

**Reply by MikNolhan** [reply]
Microservices log is only putting out that it is active and basic stuff when it starts, but then does not generate anything beyond that. It shows job concurrency changes but nothing else.

**Reply by MikNolhan** [reply]
Posted some logs to the bug page, but the issue seems to be PSQL. I am going to roll back a backup I have last night. I take physical backups every 6 hours, plus an additional logical backup every 24 hours, and that goes to AWS and through some life cycle rules. Maybe you know a way to just reset the job status there via PSQL directly?

**Comment by MikNolhan** [comment]
Logs finally started to appear, but there this is all I have here. Thumbnails do generate though. 

`tail -f /var/log/immich/immich-microservices.log`


```
[Nest] 5511  - 10/11/2024, 7:57:33 AM   ERROR [Microservices:JobService] Unable to run job handler (thumbnailGeneration/generate-thumbnails): Error: Connection terminated due to connection timeout
[Nest] 5511  - 10/11/2024, 7:57:33 AM   ERROR [Microservices:JobService] Error: Connection terminated due to connection timeout
    at Connection.<anonymous> (/media/Obelisk/Immich/App/node_modules/pg/lib/client.js:131:73)
    at Object.onceWrapper (node:events:633:28)
    at Connection.emit (node:events:519:28)
    at Socket.<anonymous> (/media/Obelisk/Immich/App/node_modules/pg/lib/connection.js:62:12)
    at Socket.emit (node:events:519:28)
    at TCP.<anonymous> (node:net:339:12)
[Nest] 5511  - 10/11/2024, 7:57:33 AM   ERROR [Microservices:JobService] Object:
{
  "id": "60c1ddff-cda6-4423-8e44-f0362b44b6d0"
}
```



```
    at async MediaService.handleGenerateThumbnails (/media/Obelisk/Immich/App/dist/services/media.service.js:101:25)
    at async /media/Obelisk/Immich/App/dist/services/job.service.js:163:36
    at async Worker.processJob (/media/Obelisk/Immich/App/node_modules/bullmq/dist/cjs/classes/worker.js:394:28)
    at async Worker.retryIfFailed (/media/Obelisk/Immich/App/node_modules/bullmq/dist/cjs/classes/worker.js:581:24)
[Nest] 5511  - 10/11/2024, 7:57:36 AM   ERROR [Microservices:JobService] Object:
{
  "id": "7fa8e7dc-2214-4866-99f6-1f369b99feef"
```

**Reply by MikNolhan** [reply]
So after more troubleshooting the issue is that it has nfact set 0 to 25. Upon changing concurrency to 1 job. The processing went to 26.
Changing to 75 it went to 100. 


The issue is when the number is a non zero value the option to click missing is gone.

**Comment by MikNolhan** [comment]
RESOLVED:

https://github.com/immich-app/immich/discussions/1896#discussioncomment-5158106

The issue was Redis, which explains the weird lack of logs, and errors mentioning that PSQL was busy even when it was entirely free and accessible by port and unix socket. I was not able to clear the issue.

`redis-cli flushall`

Optionally, you may need.

`redis-cli flushdb`
